avm/res/data-factory/factory/main.bicep

```diff
@@ -232,6 +235,23 @@ module dataFactory_integrationRuntimes 'integration-runtime/main.bicep' = [
   }
 ]
 
+module dataFactory_linkedServices 'linked-service/main.bicep' = [
+  for (linkedService, index) in linkedServices: {
+    name: '${uniqueString(deployment().name, location)}-DataFactory-LinkedServices-${index}'
+    params: {
+      dataFactoryName: dataFactory.name
+      name: linkedService.name
+      typeName: linkedService.type
```

Comment:
Similar case as with `description` originally. You expect the user to provider a value for a parameter `type`, you then take this value and assign it to a parameter `typeName` and where it's used you then translate it back to `type` 😄 I recommend to use either `typeName` all the way, or `type`. Not both 😉

Reply:
Hi @AlexanderSehr  I have made these changes now